prow/scripts/cluster-integration/kyma-upgrade-gardener-azure.sh

```diff
@@ -178,31 +181,40 @@ function getLastReleaseVersion() {
     echo "${version}"
 }
 
+function getLastReleaseCandidateVersion() {
+  version=$(curl --silent --fail --show-error "https://api.github.com/repos/kyma-project/kyma/releases?access_token=${BOT_GITHUB_TOKEN}" |
+    jq -r 'del( .[] | select( (.prerelease == false) or (.draft == true) )) | .[0].tag_name ')
+  
+  echo "${version}"
+}
+
 function installKyma() {
-    LAST_RELEASE_VERSION=$(getLastReleaseVersion)
+    #LAST_RELEASE_VERSION=$(getLastReleaseVersion)
+    # change back to release version once release 1.17 is available
+    LAST_RELEASE_VERSION=$(getLastReleaseCandidateVersion)
```

Comment:
[Kyma 1.17](https://github.com/kyma-project/kyma/releases/tag/1.17.0) is available. Please use `getLastReleaseVersion` instead of `getLastReleaseCandidateVersion`.